### PR TITLE
raw image added

### PR DESCRIPTION
**Pull request by rakeshSgr**

### CODE REVIEW

File: src/services/account.js [hunk 1]
```diff
@@ -1100,6 +1105,7 @@ module.exports = class AccountHelper {
 				await Promise.all(
 					users.data.map(async (user) => {
 						/* Assigned image url from the stored location */
+						user['image_cloud_path'] = user.image
```
Comment (aks30) [inline src/services/account.js:1108]: @rakeshSgr verify if this works and image_cloud_path is not replaced with http URL

File: src/services/account.js [hunk 2]
```diff
@@ -1062,6 +1062,11 @@ module.exports = class AccountHelper {
 					}
 				)
 
+				users.map(async (user) => {
```
Comment (aks30) [inline src/services/account.js:1065]: @rakeshSgr this can be optimised and used above

**Review by aks30 — CHANGES_REQUESTED** [review]
Reviewed on 11 NOV

File: src/services/account.js (hunk 1 above)
Comment (rakeshSgr) [reply src/services/account.js]: it won't replace

**Pull request merged**